frontend/src/helpers/validators/StaffFormValidations.ts

```diff
@@ -72,6 +72,27 @@ fieldValidations["businessInformation.lastName"] = [
   hasOnlyNamingCharacters("last name"),
 ];
 
+fieldValidations["businessInformation.businessName"] = [
+  isNotEmpty("Client name cannot be empty"),
+  isMaxSizeMsg("client name", 60),
+  isAscii("client name"),
+];
+
+fieldValidations["businessInformation.workSafeBcNumber"] = [
+  optional(isOnlyNumbers("WorkSafeBC number should contain only numbers")),
+  optional(isMaxSizeMsg("WorkSafeBC", 6))
+];
+
+fieldValidations["businessInformation.doingBusinessAs"] = [  
+  optional(isMaxSizeMsg("doing business as", 120)),
+  optional(isAscii("doing business as")),
+];
+
+fieldValidations["businessInformation.clientAcronym"] = [  
+  optional(isMaxSizeMsg("acronym", 8)),
+  optional(isAscii("acronym")),
```

Comment:
I believe this field should allow only letters and numbers.
And also it should not have less than 3 characters (unless it's blank, but the `optional` helper should be enough to take care of this).